src/templates/transaction.rs

```diff
@@ -16,6 +16,36 @@ impl PageContent for TransactionHtml {
   }
 }
 
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
```

Comment:
I think we can just derive these on the `TransactionsHtml` struct directly and then type alias like so:

```rust
pub type TransactionJson = TransactionHtml;
```

Reply:
fixed